### PR TITLE
Viz tool integration

### PR DESCRIPTION
**Pull request by mayankchetan**
## Purpose
This PR adds the WEIS output visualization tool. This was developed in support of a WEIS Q3 milestone. 

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
Currently being tested manually on Kestrel (NREL's HPS system). Need to implement an automated system. 

## Checklist
- [x] I have run existing tests which pass locally with my changes
- [ ] I have added new tests or examples that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation
- [ ] Need to solve issue reading `SQL` files introduced in #288. Temp solution implemented. @cfrontin, could use your support here.
- [x] Implement text box input on homepage to accept path to the visualization input file. @sora-ryu
- [ ] ~~Pass turbine figure object from raft to enable 3D view. @mayankchetan~~

### The Visualization tool was primarily developed by @sora-ryu

### CODE REVIEW

File: weis/visualization/utils.py [hunk 1]
```diff
@@ -452,3 +545,218 @@ def prettyprint_variables(
...
+        # Plot convergences
+        for out in opt_outs:
+            axs[out].plot(np.arange(i_plot+1),opt_outs[out][:i_plot+1],'.')
+            axs[out].set_xlabel('')
+            axs[out].set_xlim(0,n_plots)
+            axs[out].set_ylabel(out)
+
+        r = 45
+        lp = 20
+        axs['draft'].set_ylabel('Draft (m)',rotation=r, labelpad=lp)
```
Comment (dzalkind) [inline weis/visualization/utils.py:744]: We could probably skip the convergence half of the plot, since it's shown already on the page.

**Comment by yonghoonlee** [comment]
@mayankchetan I get the following error:
```
python mainApp.py --port 80 --host localhost --debug False --yaml test.yaml
/Users/yonghoonlee/work/WEIS-Visualization/weis/visualization/utils.py:601: SyntaxWarning: invalid escape sequence '\s'
  df = pd.read_csv(file_path, skiprows=[0,1,2,3,4,5,7], sep='\s+')
Traceback (most recent call last):
  File "/Users/yonghoonlee/work/WEIS-Visualization/weis/visualization/appServer/app/mainApp.py", line 47, in <module>
    dcc.Store(id='input-dict', data=parse_yaml(args.yaml)),
                                               ^^^^
NameError: name 'args' is not defined
```
Would you please take a look at this? Thank you.

**Comment by mayankchetan** [comment]
Thank you, @yonghoonlee, I'm looking into this. We are also working on updating the documentation to reflect the newer changes.

**Comment by yonghoonlee** [comment]
@mayankchetan and @sora-ryu,

I wanted to follow the README.md on the appServer but app/test.yaml file is missing. Would you please guide me how I can create this file? Thank you.

**Pull request merged**